### PR TITLE
chore: adhere to ruff's "N" rules

### PR DESCRIPTION
**Pull request by dni**
bunch of more consistent variable naming. inspired by this issue. https://github.com/lnbits/lnbits/issues/2308
WARNING this breaks some imports in extensions, UPDATE them to fix!

jmeter tests temporarly disabled https://github.com/lnbits/lnbits/issues/2410

### CODE REVIEW

**Comment by dni** [comment]
6 extensions were incompatible, pr's are open with min_version `0.12.5`

**Comment by motorina0** [comment]
> 6 extensions were incompatible, pr's are open with min_version `0.12.5`

That is why the JMeter tests fail?
Can you please add links to PRs here.

**Comment by motorina0** [comment]
JMeter test fail reasons:

```log
Could not load extension `tpos`: cannot import name 'websocketUpdater' from 'lnbits.core.services' ("lnbits/lnbits/core/services.py)
Could not load extension `cashu`: module 'cashu' has no attribute 'cashu_ext'
Could not load extension `copilot`: cannot import name 'websocketUpdater' from 'lnbits.core.services' ("lnbits/lnbits/core/services.py)
Could not load extension `discordbot`: Directory ''extensions/discordbot/static'' in package 'lnbits' could not be found.
Could not load extension `nostrrelay`: cannot import name 'websocketUpdater' from 'lnbits.core.services' ("lnbits/lnbits/core/services.py)
Could not load extension `lnurldevice`: cannot import name 'websocketUpdater' from 'lnbits.core.services' ("lnbits/lnbits/core/services.py)
Could not load extension `nostrmarket`: cannot import name 'websocketUpdater' from 'lnbits.core.services' ("lnbits/lnbits/core/services.py)
Could not load extension `bleskomat`: cannot import name 'PaymentFailure' from 'lnbits.core.services' ("lnbits/lnbits/core/services.py)
```

File: lnbits/app.py [hunk 1]
```diff
@@ -111,8 +111,8 @@ async def shutdown():
 
     # wait a bit to allow them to finish, so that cleanup can run without problems
     await asyncio.sleep(0.1)
-    WALLET = get_wallet_class()
-    await WALLET.cleanup()
+    wallet_class = get_wallet_class()
```
Comment (motorina0) [inline lnbits/app.py:114]: - the old version (`WALLET`)  follows the naming conventions for global variables: upsercase separated by underscores
- `wallet_class` is actually not a `class` but an instance of a class

Comment (dni) [reply lnbits/app.py:114]: what if we rename it to `get_funding_source` and `funding_source` on a later PR?

Comment (motorina0) [reply lnbits/app.py:114]: why on a later PR?

**Comment by dni** [comment]
https://github.com/lnbits/lnbits/issues/2409

File: lnbits/app.py (hunk 1 above)
Comment (dni) [reply lnbits/app.py]: done

**Review by motorina0 — APPROVED** [review]

File: tests/helpers.py [hunk 2]
```diff
@@ -23,19 +23,19 @@ class DbTestModel(FromRowModel):
...
 
 async def get_random_invoice_data():
     return {"out": False, "amount": 10, "memo": f"test_memo_{get_random_string(10)}"}
 
 
-set_wallet_class()
-WALLET = get_wallet_class()
+set_funding_source()
+WALLET = get_funding_source()
 is_fake: bool = WALLET.__class__.__name__ == "FakeWallet"
```
Comment (prusnak) [inline tests/helpers.py:39]: Missed this one:

```suggestion
funding_source = get_funding_source()
is_fake: bool = funding_source.__class__.__name__ == "FakeWallet"
```

**Review by prusnak — CHANGES_REQUESTED** [review]

**Review by prusnak — APPROVED** [review]

**Pull request merged**